examples/react-router/components/Nav.js

```diff
@@ -7,13 +7,14 @@ import {Link} from 'react-router';
 
 class Nav extends React.Component {
     static contextTypes = {
-        router: React.PropTypes.func.isRequired
+        history: React.PropTypes.object.isRequired,
+        location: React.PropTypes.object.isRequired
     };
     render() {
         return (
             <ul className="pure-menu pure-menu-open pure-menu-horizontal">
-                <li className={this.context.router.isActive('/') ? 'pure-menu-selected' : ''}><Link to='/'>Home</Link></li>
-                <li className={this.context.router.isActive('/about') ? 'pure-menu-selected' : ''}><Link to='/about'>About</Link></li>
+                <li className={this.context.location.pathname === '/' ? 'pure-menu-selected' : ''}><Link to='/'>Home</Link></li>
```

Comment:
Can you explain a bit on how you figure this out? I don't understand why the value of context has changed.